guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md

```diff
@@ -187,6 +187,45 @@ Note: If the array only has two elements, then the `for` loop never gets used an
 
 *   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Increment' target='_blank' rel='nofollow'>Prefix increment operator ++</a>
 
+## ![:rotating_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/rotating_light.png?v=3 ":rotating_light:") Advanced Code Solution 2:
+
+    const smallestCommons = arr => {
+      let max = Math.max(...arr);
+      let min = Math.min(...arr);
+      // Initially the solution is assigned to the highest value of the array
+      let sol = max;
+
+      for (let i = max - 1; i >= min; i--) {
+        // Each time the solution checks (i.e. sol%i===0) it won't be necessary
+        // to increment 'max' to our solution and restart the loop
+        if (sol % i) {
+          sol += max;
+          i = max;
+        }
+      }
+      return sol;
+    };
+    
+    // test here
+    smallestCommons([1,5]);
+    
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/@tiagosalema/smallestCommons' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
Hi, thank you for your PR but now we are no longer accepting links or other resources (images, etc.) which are not under our control, i.e. which a user or service might remove and is not under our control.
Please remove this link.

Reply:
Fixed on behalf of user.